docs/operation_center/integration_catalog/cloud_and_saas/google/google_kubernetes_engine.md

```diff
@@ -0,0 +1,38 @@
+uuid: 00bbde4f-cb17-4c3f-9f5e-a585fc7c8fc0
+name: Google Kubernetes Engine (GKE)
+type: intake
+
+{!operation_center/integration_catalog/cloud_and_saas/google/google_cloud.md!}
+
+{!operation_center/integration_catalog/generated/google-kubernetes-engine_do_not_edit_manually.md!}
+
+
+## Google Kubernetes Engine (GKE) configuration
+
+There's multiple kind of logs produced by GKE:
+
+**Auditd logs**: Most import logs from a security point of view. We recommend that you use [Auditbeat](../../../endpoint/auditbeat_linux/) to collect Auditd logs.
+
+**Flow Logs**: From [Google VPC FLow Logs documentation](https://cloud.google.com/vpc/docs/using-flow-logs):
+> VPC Flow Logs records a sample of network flows sent from and received by VM instances, including instances used as GKE nodes. These logs can be used for network monitoring, forensics, real-time security analysis, and expense optimization.
```

Comment:
```suggestion
> VPC Flow Logs records a sample of network flows sent from and received by VM instances, including instances used as GKE nodes. These logs can be used for network monitoring, forensics, real-time security analysis and expense optimization.
```

Reply:
This is a quote from Google's documentation